includes/admin/class-wp-job-manager-admin-notices.php

```diff
@@ -317,16 +332,40 @@ private static function generate_notice_from_updates( $updates ) {
 	 * Renders a notice.
 	 *
 	 * @param array $notice See `generate_notice_from_updates` for format.
+	 * @param string $notice_id  Unique identifier for the notice.
 	 * @return void
 	 */
-	private static function render_notice( $notice ) {
-		// TODO Handle different levels of notices.
-		echo '<div class="notice wpjm-admin-notice wpjm-admin-notice--info">';
+	private static function render_notice( $notice_id, $notice ) {
+		if ( empty( $notice['actions'] ) || ! is_array( $notice['actions'] ) ) {
+			$notice['actions'] = [];
+		}
+
+		$notice_class = '';
+		if ( ! empty( $notice['style'] ) ) {
+			$notice_class = 'wpjm-notice-' . $notice['style'];
+		}
+
+		$is_dismissible       = $notice['dismissible'];
```

Comment:
```suggestion
		$is_dismissible       = $notice['dismissible'] ?? true;
```

Not sure if true or false by default. Since false is more restrictive I would say true... but I am open for discussion.

Reply:
At the moment, `$is_dismissable` is `null`. If the endpoint sets that to an empty string it'll be `false`/`0`, but with the understanding that we want these notices to _always_ be dismissible, then I agree with defaulting to `true`. That way, a non-dismissible notice would also have to be intentional.